### ISSUE TITLE
Expected double-quoted property name in JSON

### ISSUE THREAD

**Issue by baixiaoyu2997** [DavidAnson/vscode-markdownlint#318]
info:
```
markdownlint-cli2 0.12.1
extension: markdownlint v0.53.0
```

I think this is related issue: https://github.com/DavidAnson/markdownlint-cli2/issues/267

When I use the cli it does catch errors in the md file, but not the vscode extension because of the trailing commas present in the config file

![image](https://github.com/DavidAnson/vscode-markdownlint/assets/9149999/c77afd77-ace1-49cb-89a4-f0f7ed738072)
![image](https://github.com/DavidAnson/vscode-markdownlint/assets/9149999/9dd1237f-d241-4bda-a603-bc784f042723)
 
`.markdownlint-cli2.jsonc`:
```json
{
  "$schema": "https://raw.githubusercontent.com/DavidAnson/markdownlint-cli2/main/schema/markdownlint-cli2-config-schema.json",
  "config": {
    "MD013": false,
    "MD024": false,
    "MD033": false,
    "MD041": false,
  },
  "ignores": [
    "public",
    "extends/*/public",
    ".nuxt",
    "extends/*/.nuxt",
    ".output",
    "extends/*/.output",
    "dist",
    "extends/*/dist",
    "node_modules",
    ".github",
  ],
}

```

**Comment by DavidAnson** [comment]
This is already fixed for the next release. Should be in a couple of days.

**Comment by baixiaoyu2997** [comment]
thanks!!!
